man/nanoarrow.Rd

```diff
@@ -16,13 +16,13 @@
 \alias{as_record_batch_reader.RPolarsDataFrame}
 \title{polars to nanoarrow and arrow}
 \usage{
-\method{as_nanoarrow_array_stream}{RPolarsDataFrame}(x, ..., schema = NULL)
+as_nanoarrow_array_stream.RPolarsDataFrame(x, ..., schema = NULL)
```

Comment:
Unnecessary changes...

Reply:
That's automatically made by devtools::document, I don't know why it changes that, I have R and packages up to date

Reply:
Oh, I didn't know you didn't know this.
We have to load some packages before updating the Rd file or it will not be recognized some funtions as S3 methods.

https://github.com/pola-rs/r-polars/blob/870cb3ac7b7aa144b5d67f2a184eb6e61de7e3be/R/zzz.R#L148-L153

This is one of the reasons why we have to use Taskfile for development.

https://github.com/pola-rs/r-polars/blob/870cb3ac7b7aa144b5d67f2a184eb6e61de7e3be/Taskfile.yml#L10
https://github.com/pola-rs/r-polars/blob/870cb3ac7b7aa144b5d67f2a184eb6e61de7e3be/Taskfile.yml#L21
https://github.com/pola-rs/r-polars/blob/870cb3ac7b7aa144b5d67f2a184eb6e61de7e3be/Taskfile.yml#L177-L180

Reply:
thanks, I'll load those in the Rprofile then, that should solve it